desc/equilibrium/equilibrium.py

```diff
@@ -364,6 +389,11 @@ def __init__(
             ValueError,
             "Surface and Equilibrium must have the same symmetry",
         )
+        errorif(
```

Comment:
I think this should get checked above where the other symmetry stuff is?

Reply:
What do you mean by this?